cpu/em_smt_folding_test.py

```diff
@@ -71,7 +73,41 @@ def test(self):
         1. Disable all the idle states
         2. Run ebizzy when smt=off and smt=on
         3. Enable all the idle states.
+        4. run cpu Dlpar operation in parallel
+        '''
+        workload_thread = Thread(target=self.run_workload)
+        workload_thread.start()
+        dlpar_thread = Thread(target=self.dlpar_cpu_hotplug)
+        dlpar_thread.start()
+        workload_thread.join()
+        dlpar_thread.join()
+
+    def run_ebizzy(self):
+        '''
+        Run ebizzy by doing taskset
         '''
+        output = process.system_output("taskset -c %s ./ebizzy -t1"
+                                       " -S 6 -s 4096" % self.cpu, shell=True)
+        return output.split()[0]
+
+    def dlpar_cpu_hotplug(self):
+        if 'ppc' in platform.processor() and 'PowerNV' not in open('/proc/cpuinfo', 'r').read():
```

Comment:
in `setUp` `ppc` part was already checked and it's not required here.
```
        if 'ppc' not in distro.detect().arch:
            self.cancel("Processor is not ppc64")
```
and for reading `/proc/cpuinfo` please use `genio.read_file`

Reply:
@nasastry ah sure , will make changes and push